build/templates/grpc_session_options.rst.mako

```diff
@@ -0,0 +1,101 @@
+<%
+    import build.helper as helper
+
+    config = template_parameters['metadata'].config
+    module_name = config['module_name']
+    driver_name = config['driver_name']
+    enums = config['enums']
+%>\
+${helper.get_rst_header_snippet('gRPC Support', '=')}
+
+Support for using ${driver_name} over gRPC
+
+.. py:currentmodule:: ${module_name}
+
+
+Enums
+=====
```

Comment:
This is showing up in the TOC for some reason.
It's kind of awkward because it doesn't show up under a toggle for gRPC Support.

image.png

Reply:
Based on https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections, maybe replace ==== with ----, ---- with ^^^^^

Reply:
Since there's only 1 enum and 1 class, I got rid of those headers and just made the 2 entries be the subsections. It seems nice to me.